torchvision/csrc/io/image/cpu/decode_jpeg.cpp

```diff
@@ -191,6 +198,54 @@ torch::Tensor decode_jpeg(const torch::Tensor& data, ImageReadMode mode) {
     jpeg_calc_output_dimensions(&cinfo);
   }
 
+  int exif_orientation = 0;
```

Comment:
Should we just move this entire implementation as a function within `exif.h`? It might make it easier w.r.t. copyright, e.g. we would only need to copy the copyright notice in a single file.